File: SemantikosClientesWS/ClienteJava/src/cl/minsal/semantikos/ws/shared/IndirectCrossmap.java

package cl.minsal.semantikos.ws.shared;

import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorType;
import javax.xml.bind.annotation.XmlType;


/**
 * <p>Java class for IndirectCrossmap complex type.
 * 
 * <p>The following schema fragment specifies the expected content contained within this class.
 * 
 * <pre>
 * &lt;complexType name="IndirectCrossmap">
 *   &lt;complexContent>
 *     &lt;restriction base="{http://www.w3.org/2001/XMLSchema}anyType">
 *       &lt;sequence>
 *         &lt;element name="mapGroup" type="{http://www.w3.org/2001/XMLSchema}int"/>
 *         &lt;element name="mapPriority" type="{http://www.w3.org/2001/XMLSchema}int"/>
 *         &lt;element name="mapRule" type="{http://www.w3.org/2001/XMLSchema}string" minOccurs="0"/>
 *         &lt;element name="mapAdvice" type="{http://www.w3.org/2001/XMLSchema}string" minOccurs="0"/>
 *         &lt;element name="mapTarget" type="{http://www.w3.org/2001/XMLSchema}string" minOccurs="0"/>
 *         &lt;element name="correlation" type="{http://www.w3.org/2001/XMLSchema}long"/>
 *         &lt;element name="idCrossmapCategory" type="{http://www.w3.org/2001/XMLSchema}long"/>
 *         &lt;element name="state" type="{http://www.w3.org/2001/XMLSchema}boolean"/>
 *       &lt;/sequence>
 *     &lt;/restriction>
 *   &lt;/complexContent>
 * &lt;/complexType>
 * </pre>
 * 
 * 
 */
@XmlAccessorType(XmlAccessType.FIELD)
@XmlType(name = "IndirectCrossmap", propOrder = {
    "mapGroup",
    "mapPriority",
    "mapRule",
    "mapAdvice",
    "mapTarget",
    "correlation",
    "idCrossmapCategory",
    "state"
})
public class IndirectCrossmap {

    protected int mapGroup;
    protected int mapPriority;
    protected String mapRule;
    protected String mapAdvice;
    protected String mapTarget;
    protected long correlation;
    protected long idCrossmapCategory;
    protected boolean state;

    /**
     * Gets the value of the mapGroup property.
     * 
     */
    public int getMapGroup() {
        return mapGroup;
    }

    /**
     * Sets the value of the mapGroup property.
     * 
     */
    public void setMapGroup(int value) {
        this.mapGroup = value;
    }

    /**
     * Gets the value of the mapPriority property.
     * 
     */
    public int getMapPriority() {
        return mapPriority;
    }

    /**
     * Sets the value of the mapPriority property.
     * 
     */
    public void setMapPriority(int value) {
        this.mapPriority = value;
    }

    /**
     * Gets the value of the mapRule property.
     * 
     * @return
     *     possible object is
     *     {@link String }
     *     
     */
    public String getMapRule() {
        return mapRule;
    }

    /**
     * Sets the value of the mapRule property.
     * 
     * @param value
     *     allowed object is
     *     {@link String }
     *     
     */
    public void setMapRule(String value) {
        this.mapRule = value;
    }

    /**
     * Gets the value of the mapAdvice property.
     * 
     * @return
     *     possible object is
     *     {@link String }
     *     
     */
    public String getMapAdvice() {
        return mapAdvice;
    }

    /**
     * Sets the value of the mapAdvice property.
     * 
     * @param value
     *     allowed object is
     *     {@link String }
     *     
     */
    public void setMapAdvice(String value) {
        this.mapAdvice = value;
    }

    /**
     * Gets the value of the mapTarget property.
     * 
     * @return
     *     possible object is
     *     {@link String }
     *     
     */
    public String getMapTarget() {
        return mapTarget;
    }

    /**
     * Sets the value of the mapTarget property.
     * 
     * @param value
     *     allowed object is
     *     {@link String }
     *     
     */
    public void setMapTarget(String value) {
        this.mapTarget = value;
    }

    /**
     * Gets the value of the correlation property.
     * 
     */
    public long getCorrelation() {
        return correlation;
    }

    /**
     * Sets the value of the correlation property.
     * 
     */
    public void setCorrelation(long value) {
        this.correlation = value;
    }

    /**
     * Gets the value of the idCrossmapCategory property.
     * 
     */
    public long getIdCrossmapCategory() {
        return idCrossmapCategory;
    }

    /**
     * Sets the value of the idCrossmapCategory property.
     * 
     */
    public void setIdCrossmapCategory(long value) {
        this.idCrossmapCategory = value;
    }

    /**
     * Gets the value of the state property.
     * 
     */
    public boolean isState() {
        return state;
    }

    /**
     * Sets the value of the state property.
     * 
     */
    public void setState(boolean value) {
        this.state = value;
    }

}

File: SemantikosClientesWS/clientePHP/src/Semantikos/ClientBundle/API/IndirectCrossmap.php
<?php

namespace Semantikos\ClientBundle\API;

class IndirectCrossmap
{

    /**
     * @var int $mapGroup
     */
    protected $mapGroup = null;

    /**
     * @var int $mapPriority
     */
    protected $mapPriority = null;

    /**
     * @var string $mapRule
     */
    protected $mapRule = null;

    /**
     * @var string $mapAdvice
     */
    protected $mapAdvice = null;

    /**
     * @var string $mapTarget
     */
    protected $mapTarget = null;

    /**
     * @var int $correlation
     */
    protected $correlation = null;

    /**
     * @var int $idCrossmapCategory
     */
    protected $idCrossmapCategory = null;

    /**
     * @var boolean $state
     */
    protected $state = null;

    /**
     * @param int $mapGroup
     * @param int $mapPriority
     * @param int $correlation
     * @param int $idCrossmapCategory
     * @param boolean $state
     */
    public function __construct($mapGroup, $mapPriority, $correlation, $idCrossmapCategory, $state)
    {
      $this->mapGroup = $mapGroup;
      $this->mapPriority = $mapPriority;
      $this->correlation = $correlation;
      $this->idCrossmapCategory = $idCrossmapCategory;
      $this->state = $state;
    }

    /**
     * @return int
     */
    public function getMapGroup()
    {
      return $this->mapGroup;
    }

    /**
     * @param int $mapGroup
     * @return IndirectCrossmap
     */
    public function setMapGroup($mapGroup)
    {
      $this->mapGroup = $mapGroup;
      return $this;
    }

    /**
     * @return int
     */
    public function getMapPriority()
    {
      return $this->mapPriority;
    }

    /**
     * @param int $mapPriority
     * @return IndirectCrossmap
     */
    public function setMapPriority($mapPriority)
    {
      $this->mapPriority = $mapPriority;
      return $this;
    }

    /**
     * @return string
     */
    public function getMapRule()
    {
      return $this->mapRule;
    }

    /**
     * @param string $mapRule
     * @return IndirectCrossmap
     */
    public function setMapRule($mapRule)
    {
      $this->mapRule = $mapRule;
      return $this;
    }

    /**
     * @return string
     */
    public function getMapAdvice()
    {
      return $this->mapAdvice;
    }

    /**
     * @param string $mapAdvice
     * @return IndirectCrossmap
     */
    public function setMapAdvice($mapAdvice)
    {
      $this->mapAdvice = $mapAdvice;
      return $this;
    }

    /**
     * @return string
     */
    public function getMapTarget()
    {
      return $this->mapTarget;
    }

    /**
     * @param string $mapTarget
     * @return IndirectCrossmap
     */
    public function setMapTarget($mapTarget)
    {
      $this->mapTarget = $mapTarget;
      return $this;
    }

    /**
     * @return int
     */
    public function getCorrelation()
    {
      return $this->correlation;
    }

    /**
     * @param int $correlation
     * @return IndirectCrossmap
     */
    public function setCorrelation($correlation)
    {
      $this->correlation = $correlation;
      return $this;
    }

    /**
     * @return int
     */
    public function getIdCrossmapCategory()
    {
      return $this->idCrossmapCategory;
    }

    /**
     * @param int $idCrossmapCategory
     * @return IndirectCrossmap
     */
    public function setIdCrossmapCategory($idCrossmapCategory)
    {
      $this->idCrossmapCategory = $idCrossmapCategory;
      return $this;
    }

    /**
     * @return boolean
     */
    public function getState()
    {
      return $this->state;
    }

    /**
     * @param boolean $state
     * @return IndirectCrossmap
     */
    public function setState($state)
    {
      $this->state = $state;
      return $this;
    }

}
